flow/probes/ebpf.go

```diff
@@ -439,6 +519,7 @@ func (p *EBPFProbesHandler) registerProbe(n *graph.Node, capture *types.Capture,
 		if err := elf.DetachSocketFilter(socketFilter, fd); err != nil {
 			logging.GetLogger().Errorf("Unable to detach eBPF probe: %s", err)
 		}
+		rs.Close()
```

Comment:
This should also be done line 492 (for return fmt.Errorf....)

Reply:
good catch